feature/metis/shared/src/main/kotlin/de/tum/informatics/www1/artemis/native_app/feature/metis/shared/db/entities/BasePostingEntity.kt

```diff
@@ -36,7 +36,9 @@ data class BasePostingEntity(
     @ColumnInfo(name = "content")
     val content: String?,
     @ColumnInfo(name = "author_role")
-    val authorRole: UserRole?
+    val authorRole: UserRole?,
+    @ColumnInfo(name = "author_image_url")
+    val authorImageUrl: String?,
```

Comment:
This field should not be necessary as you already added it to the metis user entity.

Reply:
Thanks for catching that one. I found that the reactions currently miss the author imageUrl, which messes with updating the `users` database table. I create a PR to fix this in the Artemis repo: https://github.com/ls1intum/Artemis/pull/9897